pages/install/smelt.hbs

```diff
@@ -0,0 +1,33 @@
+---
+title: Moto 360 2015
+deviceName: smelt
+section: install
+layout: aw-install
+---
+
+<table width="100%"><thead><tr>
+<th>Display</th>
+<th>Touch</th>
+<th>Microphone</th>
+<th>Bluetooth</th>
+<th>Haptics</th>
+<th>USB</th>
+<th>WLAN</th>
+<th>Heart Rate</th>
+<th>Tilt-to-Wake</th>
+</tr></thead><tbody><tr height="25px">
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#b94a48"></td>
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#5ab948"></td>
+<td bgcolor="#5ab948"></td>
+</tr></tbody></table>
+
+<div class="callout callout-info">
+    <h4>USB connection</h4>
+    <p>The Moto 360 2015 does not expose a USB connection. <a href="https://github.com/AsteroidOS/meta-smelt-hybris/blob/master/README-system-dir.md#0-connect-the-watch-via-usb">You will need to solder a USB connector or create an adapter for it.</a></p>
```

Comment:
Can't you inline the content of that README in the webpage?

Reply:
Sure I can do that too. Do you want me to include the images as well?